crates/rooch-framework/sources/account.move

```diff
@@ -91,6 +96,30 @@ module rooch_framework::account{
       new_account
    }
 
+   /// Updates an existing `Account` resource under `existing_address`. A signer representing `existing_address`
+   /// is returned. This way, the caller of this function can publish additional resources under
+   /// `existing_address`.
+   public(friend) fun update_account(ctx: &mut StorageContext, existing_address: address): signer {
+      // there should be an Account resource under existing address already.
+      assert!(
+         account_storage::global_exists<Account>(ctx, existing_address),
+         error::not_found(EAccountNotExist)
+      ); 
+
+      update_account_unchecked(ctx, existing_address)
+   }
+
+   // TODO: update account should handle logics like installing auth validator with auth register
+   // for non-built-in custom validator scheme
+   fun update_account_unchecked(ctx: &mut StorageContext, existing_address: address): signer {
```

Comment:
We do not need a `update_account` function to install auth validator.`account_authentication::install_auth_validator` can install the validator, and the validator should call `account_authentication::rotate_authentication_key` to update the auth key.

Reply:
So I could call install_auth_validator to register an auth validator if it doesn't exist in BuiltInScheme?

What if I call with register with an existing BuiltInScheme? Will it result EAuthValidatorAlreadyInstalled?

From my understanding, all built in validators are registered in genesis init, so there's no need to handle updates for  BuiltInSchemes, and I could remove `.sign_and_execute` for account updates?

Reply:
If I replace `.sign_and_execute` with Ok(), it will not validate the key pairs added from this command.

Reply:
The BuiltInScheme does not need to register or install_auth_validator. Just call `rotate_authentication_key` to update the user's auth key for the validator.

Reply:
Okay. I'll learn how to do it.

For example, how to pass the private_generics of ValidatorType? Pass through function_name like `Identifier::new("rotate_authentication_key<ValidatorType>").unwrap()` and define a ValidatorType in the same Move module?

Reply:
Yes, like `rooch_framework::ed25519_validator::Ed25519Validator`

Reply:
Got it.

Reply:
I've added rotate auth key function for update.rs.